Code/FooBar.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+class FooBar
+{
+    public function checker($number)
+    {
+        if (is_int($number) && $number > 0) {
```

Comment:
I would suggest to throw exception right away. Or function body should be so small, that I dont need to scroll to get to "else".